.github/workflows/sql-test.yml

```diff
@@ -6,7 +6,34 @@ on:
       - ".github/workflows/sql-test.yml"
 
 jobs:
+  version_increment:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Instead of duplicating `version_increment` in both `sql-test` and `sql-version-check`, can you have it just in `sql-version-check` and then make the action run for both `pull_request` AND `pull_request_review` triggers?